src/hashTable.js

```diff
@@ -0,0 +1,125 @@
+class Node {
+  constructor(key, value) {
+    this.key = key,
+    this.value = value,
+    this.next = null
+  }
+}
+
+export default class HashTable {
+  constructor() {
+    this._length = 0
+    this.valueStore = []
+  }
+
+  hashFunction(key) {
+    let sum = 0
+    for (let i = 0; i < key.length; i++) {
+      sum += key.charCodeAt(i)
+    }
+    return sum % 3550
```

Comment:
Why `3550`? The use of an intention revealing constant, instead of a "magic number", might help to make this code clearer.